src/app/medical-record/medical-record.component.html

```diff
@@ -98,39 +114,44 @@ <h3 class="Color--warn Line--Break">Participant was withdrawn from the study!</h
       <br/>
     </div>
     <div class="Float--left Width--30" *ngIf="participant.participant != null && !participantExited">
-      <button md-raised-button color="primary"
-              [disabled]="isCoverDownloadDisabled()"
-              (click)="downloadCoverPDFs()">Download cover pdf
-      </button>
-      <a href="#" (click)="doNothing()" *ngIf="!isCoverDownloadDisabled()" class="Router--Outlet"><b> Change request
-        settings </b></a>
-      <div *ngIf="isCoverDownloadDisabled()"><b>* Institution name is needed for download</b></div>
-      <br/>
-      <br/>
       <div *ngIf="pdfs != null && pdfs.length != 0 && getRole().allowedToDownloadPDF()">
-        <md-select placeholder="Select PDF" class="Input--Abstraction" [(ngModel)]="selectedPDF">
-          <md-option *ngFor="let e of pdfs" [value]="e.configName">
-            {{e.displayName}}
-          </md-option>
-        </md-select>
-        <br/>
-        <br/>
-        <button md-raised-button color="primary"
-                [disabled]="selectedPDF == null || selectedPDF === '' || disableDownloadConsent"
-                (click)="downloadPDFs(selectedPDF)">Download selected PDF
-        </button>
-      </div>
-      <div *ngIf="participant.data.dsm['pdfs'] == null">
-        <button md-raised-button color="primary"
-                [disabled]="disableDownloadConsent"
-                (click)="downloadConsentPDFs()">Download consent pdf
-        </button>
-        <br/>
-        <br/>
-        <button md-raised-button color="primary"
-                [disabled]="disableDownloadRelease"
-                (click)="downloadReleasePDFs()">Download release pdf
-        </button>
+        <tr>
+          <td>
+            <md-select placeholder="Select PDF" class="Input--Abstraction" [(ngModel)]="selectedPDF">
+              <md-option *ngFor="let e of pdfs" [value]="e.configName">
+                {{e.displayName}}
+              </md-option>
+            </md-select>
+          </td>
+          <td>
+            <button md-raised-button color="primary"
+                    [disabled]="medicalRecord.name == null || medicalRecord.name === '' || downloading"
+                    (click)="downloadPDF(null)">Download PDF Bundle
+            </button>
+          </td>
+        </tr>
+        <tr>
+          <td>
+            <div *ngIf="(medicalRecord.name == null || medicalRecord.name === '')"><b>* Institution name is needed for download</b></div>
```

Comment:
In Javascript checking empty, null or undefined value can be done just using `!medicalRecord.name`.